data/CosmosDbSeeder/items/local/index_data/autotest_booking_01_index.json

```diff
@@ -0,0 +1,10 @@
+{
+  "reference": "autotest-booking-reference-01",
```

Comment:
This reference (and id) want updating to match the now renamed booking 

Also I think we should add index documents for bookings 16 through 19 also